tests/test-errors.c

```diff
@@ -11,7 +18,18 @@ int testElfCreationError() {
 int testTmpCreationError() {
   SDTProvider_t *provider;
   SDTError_t errno;
+#ifdef HAVE_LIBSTAPSDT_MEMORY_BACKED_FD
+  char *providerLongName = "test/probe/creation/error/with/a/name/longer/than/"
```

Comment:
the path name used to fail the test for tempfiles is actually valid for memfd, so we force it to fail by sending it a name that is too long.